mobify/platform-scaffold
# [WEB-1392] Delay Default Ask to second page view

In an attempt to ~~reduce the number network calls (faster to downloading everything) and~~ avoid extraneous renders (faster visually complete time), we've delayed the default ask to only show on the second page view.

 **JIRA**: [WEB-1392](https://mobify.atlassian.net/browse/WEB-1392)

## Changes
- Change `showOnPageCount` value from `1` to `2`

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Refresh the page
- *Verify* that the second page load caused the DefaultAsk to show
